tutorials/fundamentals/typeof-date.md

```diff
@@ -0,0 +1,28 @@
+JavaScript date's are considered objects. As a result,
+using `typeof` would result in it returing a string, `'object'`.
+The problem with this however, is that objects and nulls would pass
+any checks as well. To perform a more exact check, you should instead use
+the `Object.prototype.toString.call(variableToCheck)` method. It returns the internal class
```

Comment:
The `[object Date]` approach is valid, but I find it to be a bit cumbersome. I'd instead recommend using [`v instanceof Date`](https://masteringjs.io/tutorials/fundamentals/instanceof).